tutorials/question_generation.py

```diff
@@ -0,0 +1,53 @@
+from transformers import pipeline
```

Comment:
All other tutorials are numbered so let's rename this file to `Tutorial13_Question_Generation.py`.